### PR TITLE
Add React style section to README

### PR DESCRIPTION
**Pull request by ericqweinstein**
^

CC @pgoldrbx, @rufus2021

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1448,6 +1449,52 @@
 **[⬆ back to top](#table-of-contents)**
 
 
+## React
+Our React style is mostly influenced by [David Chang's style guide](https://reactjsnews.com/react-style-guide-patterns-i-like/). Exceptions are enumerated below.
+
+  - Since `displayName` is automatically set by React when calling `React.createClass()`, there's no need to explicitly include one:
```
Comment (pgoldrbx) [inline README.md]: @ericqweinstein - is it set when calling `createClass` or is it created during the transpile phase?

File: README.md [hunk 2]
```diff
@@ -1448,6 +1449,52 @@
...
+
+  ```javascript
+    renderContent: function () {
+      var content =
+        <div>
+          {this.props.rubric && <a className='rubric' href={this.props.rubricUrl} dangerouslySetInnerHTML={{__html: this.props.rubric}} />}
+          {this.props.hed && <div className='hed' dangerouslySetInnerHTML={{__html: this.props.hed}} />}
+          {this.props.dek && <div className='dek' dangerouslySetInnerHTML={{__html: this.props.dek}} />}
+          {this.props.date && <div className='date'>{this.props.date}</div>}
+        </div>;
```
Comment (pgoldrbx) [inline README.md]: We could give some other examples here

```jsx
{this.props.isTruthy && <MyComponent />}
{this.props.isTruthy && this.renderMyComponent()}
```

File: README.md (hunk 1 above)
Comment (ericqweinstein) [inline README.md]: @pgoldrbx Good point—will check.

File: README.md (hunk 2 above)
Comment (ericqweinstein) [inline README.md]: I'll include!

**Comment by ericqweinstein** [comment]
@pgoldrbx Updated!

**Comment by pgoldrbx** [comment]
looks good

**Pull request merged**